pkg/apimachinery/utils/meta.go

```diff
@@ -17,6 +17,18 @@ import (
 	"k8s.io/apimachinery/pkg/types"
 )
 
+// LabelKeyGetHistory is used to select object history for an given resource
+const LabelKeyGetHistory = "grafana.app/get-history"
```

Comment:
We probably want to prevent storing resources with those reserved labels ?

Reply:
Updated with a check and test -- good catch

Reply:
Spin-balling. How does it work when we restore a dashboard ? Do we re-apply from the deleted value ... or do we re-fetch the dashboard at the desired revision ?